src/GeositeFramework/js/BasemapSelector.js

```diff
@@ -15,6 +15,9 @@
                hideBasemapList(view, e);
            }
         });
+        $('#help-overlay-start').on('click', function(e) {
+            hideBasemapList(view, e);
```

Comment:
Wasn't sure either -- I think something about how that thing in particular is set up may be intercepting click events on `document`. I'd guess that would happen because it's essentially placing an overlay on top of the whole page with elements which may stop clicks from propagating through?